### PR TITLE
[tests-only] [full-ci] remove more closed issue tags from features and expected failures

### PR DESCRIPTION
**Pull request by kiranparajuli589**
## Description
- Found and removed/replaced closed issues inside feature files and expected failure files:
    - https://github.com/owncloud/ocis-reva/issues/243
    - https://github.com/owncloud/ocis-reva/issues/39 ----> https://github.com/owncloud/ocis/issues/1330
    - https://github.com/owncloud/ocis-reva/issues/64
    - https://github.com/owncloud/ocis-reva/issues/41 ----> https://github.com/owncloud/ocis/issues/1328
    - https://github.com/owncloud/ocis/issues/1743
    - https://github.com/owncloud/ocis/issues/730
    - https://github.com/owncloud/web/issues/4193
    - https://github.com/owncloud/web/issues/4171

- ~~Fixed issue headers in expected failure files~~ done https://github.com/owncloud/web/pull/5505

## Motivation and Context
do not use closed issue ref inside features or expected failure files

## How Has This Been Tested?

- 👨🏼‍🎨 
- 🤖 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [x] Tests

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] ...

### CODE REVIEW

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalGroupsSharingIndicator</strong> https://drone.owncloud.com/owncloud/web/17113/22/1
The following scenarios passed on retry:
- webUISharingInternalGroupsToRootSharingIndicator/shareWithGroups.feature:104

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17113/47/1
The following scenarios passed on retry:
- webUISharingAcceptShares/acceptShares.feature:100
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISResharing1</strong> https://drone.owncloud.com/owncloud/web/17113/57/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17114/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISResharing1</strong> https://drone.owncloud.com/owncloud/web/17114/57/1
:boom: The acceptance tests pipeline failed. The build has been cancelled.

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17117/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oCISSharingBasic</strong> https://drone.owncloud.com/owncloud/web/17122/47/1
The following scenarios passed on retry:
- webUIRestrictSharing/restrictSharing.feature:49

**Comment by ownclouders** [comment]
Results for <strong>oC10SharingInternalUsers</strong> https://drone.owncloud.com/owncloud/web/17122/23/1
The following scenarios passed on retry:
- webUISharingInternalUsersShareWithPage/shareWithUsers.feature:91

**Review by Talank — COMMENTED** [review]
lgtm

**Review by SwikritiT — COMMENTED** [review]
LGTM

**Comment by phil-davis** [comment]
@kiranparajuli589 I merged another commit-id-update PR just now, so you have conflicts - sorry!

Please rebase and resolve.

File: tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md [hunk 1]
```diff
@@ -419,6 +420,55 @@ Other free text and markdown formatting can be used elsewhere in the document if
 -   [webUISharingInternalUsers/shareWithUsers.feature:360](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingInternalUsers/shareWithUsers.feature#L360)
 -   [webUISharingPublicBasic/publicLinkCreate.feature:190](https://github.com/owncloud/web/blob/master/tests/acceptance/features/webUISharingPublicBasic/publicLinkCreate.feature#L190)
 
+### [Admin settings feature not implemented yet](https://github.com/owncloud/web/issues/5015)
```
Comment (phil-davis) [inline tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md:423]: This test suite was deleted last week. IMO we do not need to addd these epected-failures.

Comment (kiranparajuli589) [reply tests/acceptance/expected-failures-with-ocis-server-ocis-storage.md]: sorry, I was not aware of that.
removed!

**Review by phil-davis — APPROVED** [review]
LGTM if CI passes

**Comment by kiranparajuli589** [comment]
> LGTM if CI passes

there will be some line changes, I've removed some more closed issue tags. should be fixed in next commit

**Comment by phil-davis** [comment]
> > LGTM if CI passes
> 
> there will be some line changes, I've removed some more closed issue tags. should be fixed in next commit

Yes, needs some line number changes:
https://drone.owncloud.com/owncloud/web/17272/54/9
https://drone.owncloud.com/owncloud/web/17272/55/9

It will be good to get this sorted out. Then we can make another commit id update in OCIS to bring in these changes...

**Comment by phil-davis** [comment]
@kiranparajuli589 one more line number to change: https://drone.owncloud.com/owncloud/web/17280/54/9

**Review by phil-davis — APPROVED** [review]

**Pull request merged**